src/main/kotlin/no/nav/familie/ef/sak/fagsak/SøkController.kt

```diff
@@ -27,6 +28,13 @@ class SøkController(private val søkService: SøkService, private val tilgangSe
         return Ressurs.success(søkService.søkPerson(personIdentRequest.personIdent))
     }
 
+    @PostMapping("/uten-fagsak")
+    fun søkPersonUtenFagsak(@RequestBody personIdentRequest: PersonIdentDto): Ressurs<SøkeresultatUtenFagsak> {
```

Comment:
Sett i retrospekt skulle jeg ønske vi hadde `/person` i api-et for `søkPerson`. 
Og særlig nå som vi skal søke uten fagsak.

Forslag:
Legg inn to url-mappinger i `søkPerson` -> `["", "/person"]`
Endre denne til `/person/uten-fagsak`

Reply:
Fikser ✅